dts/bindings/serial/st,stm32-lpuart.yaml

```diff
@@ -18,4 +18,3 @@ properties:
       required: false
       type: int
       description: Configures the parity of the adapter. Value 0 for none, 1 for odd and 2 for even parity.
```

Comment:
Should we state something like 'defaults to None when missing' ?

Reply:
updated to description